lib/rpc/rpc.dart

```diff
@@ -56,6 +58,24 @@ class RPCLive implements RPC {
     return formatted as String;
   }
 
+  @override
+  Future<String> getRefundAddress() async {
+    var address = await _client.call('getnewaddress', ['Sidechain Deposit', 'Sidechain withdrawal refund']) as String;
```

Comment:
This is incorrect. The first parameter to `getnewaddress` is a label to apply to the address. The second parameter is the address type. This need to be set to `legacy`, I believe

Reply:
yes, this is a mistake on my part, oops